testsuite/integration/microprofile/pom.xml

```diff
@@ -503,6 +503,7 @@
                         <TC_LOGGING>true</TC_LOGGING>
                     </environmentVariables>
                     <systemPropertyVariables>
+                        <testcontainers.reuse.enable>false</testcontainers.reuse.enable>
```

Comment:
What's the purpose of this?

Reading https://java.testcontainers.org/features/reuse/ - 'Those containers won't stop after all tests are finished.' so this is definitely disabled by default, so I am not sure why this is desired here.

Reply:
👍🏼